modules/ecs/variables.tf

```diff
@@ -336,3 +336,9 @@ variable "kong_portal_api_url" {
   description = "The Portal API URL of the Portal."
   type        = string
 }
+
+variable "kong_plugins" {
+  description = "Comma-separated list of Kong plugins load, passed through the variable KONG_PLUGINS"
```

Comment:
Should this be `Comma-separated list of Kong plugins loaded` ?

Reply:
Yep 👍